README.md

```diff
@@ -304,7 +304,8 @@ We use the [companion branch approach](https://github.com/DroidKaigi/conference-
 
 <img src="https://user-images.githubusercontent.com/1386930/236188326-ddd617ae-b216-476c-9d92-e36ad02a2670.png" width="600" />
 
-* Why not GitHub Actions Artifacts, Git LFS, or Feature Branch Commits?
+> [!NOTE]
+> Why not GitHub Actions Artifacts, Git LFS, or Feature Branch Commits?
```

Comment:
Well, I believe 'note' is intended for annotations like this, rather than serving as a title 👀 
https://github.com/android/nowinandroid/tree/main#baseline-profiles